File: BackEnd/src/Model/Entities/Categoria.java
package hospiolot.model.business.entities;

import hospiolot.model.business.entities.Entity;

public class Categoria extends Entity {

    private String nom;

    public Categoria(String nom) {
        this.nom = nom;
    }
    

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    @Override
    public String toString() {
        return nom ;
    }

}

File: BackEnd/src/Model/Entities/Categoria.ts
class Categoria {

    private nom: string;

    public constructor(nom: string) {
        this.nom = nom;
    }

    public getNom(): string {
        return this.nom;
    }

    public setNom(nom: string): void {
        this.nom = nom;
    }

    public toString(): string {
        return this.nom;
    }
}

export = Categoria;